src/Cookbook.js

```diff
@@ -0,0 +1,29 @@
+const ingredientInfo = require('../data/ingredients').ingredientsData;
+
+class Cookbook {
+  constructor(allRecipes) {
+    this.allRecipes = allRecipes;
+  };
+
+  filterAllRecipesByTag(tag) {
+    return this.allRecipes.reduce((foundRecipes, recipe) => {
+			if (recipe.tags.includes(tag)) {
+				foundRecipes.push(recipe);
+			};
+			return foundRecipes;
+		}, []);
+  };
+
+  filterAllRecipesByIngredient(item) {
+    const inputtedItemID = (ingredientInfo.find(ingredient => ingredient.name.includes(item))).id;
```

Comment:
Nicole - great catch on using includes here so that the user can see results that have only part of the ingredient name!